deephaven_ipywidgets/deephaven.py

```diff
@@ -29,11 +29,17 @@ def _path_for_object(obj):
     """Return the iframe path for the specified object. Inspects the class name to determine."""
     name = _str_object_type(obj)
 
-    if name in ('deephaven.table.Table', 'pandas.core.frame.DataFrame', 'pydeephaven.table.Table'):
-        return 'table'
-    if name == 'deephaven.plot.figure.Figure':
-        return 'chart'
-    raise TypeError(f"Unknown object type: {name}")
+    if name in (
+        "deephaven.table.Table",
+        "pandas.core.frame.DataFrame",
+        "pydeephaven.table.Table",
+    ):
```

Comment:
Well we're cleaning things up, we should move this tuple to a static set variable instead